device-base/generate-dockerfile.sh

```diff
@@ -21,21 +35,34 @@ for device in $devices; do
 	;;
 	esac
 
-	for suite in $suites; do		
-
+	for suite in $suites; do
 		dockerfilePath="$device"
 		mkdir -p $dockerfilePath/$suite
-		sed -e s~#{FROM}~resin/$baseImage:$suite~g \
-			-e s~#{SUITE}~$suite~g $template > $dockerfilePath/$suite/Dockerfile
 
-	done
+		if [ $device == 'beaglebone' ]; then
 
-	# Only for armv7hf devices
-	if [ $device == 'raspberrypi2' ] || [ $device == 'beaglebone' ]; then
-		mkdir -p $dockerfilePath/sid
-		sed -e s~#{FROM}~resin/$baseImage:sid~g \
-			-e s~#{SUITE}~sid~g $template > $dockerfilePath/sid/Dockerfile		
-	fi
+			case "$suite" in
+			'wheezy')
+				sourcelist="$bb_sourceslist_wheezy_cmd"
```

Comment:
Oops, I get it wrong, it works. Btw I agree that this change won't make it look better.